appcenter-crashes/scripts/postlink.js

```diff
@@ -1,22 +1,53 @@
 const rnpmlink = require('appcenter-link-scripts');
-const npmPackages = require('./../package.json');
+const inquirer = require('inquirer');
 
 return rnpmlink.ios.checkIfAppDelegateExists()
     .then(() => rnpmlink.ios.initAppCenterConfig().catch((e) => {
         console.log(`Could not create or update AppCenter config file (AppCenter-Config.plist). Error Reason - ${e.message}`);
         return Promise.reject();
     }))
     .then(() => {
-        const prompt = npmPackages.rnpm.params[0];
-        prompt.message = prompt.message.replace(/Android/, 'iOS');
-        return rnpmlink.inquirer.prompt(prompt)
-            .catch((e) => {
-                console.log(`Could not determine when to send AppCenter crashes. Error Reason - ${e.message}`);
-                return Promise.reject();
-            });
+        return inquirer.prompt([{
+            type: 'list',
+            name: 'whenToSendCrashes',
+            message: 'For the Android app, should crashes be sent automatically or processed in JavaScript before being sent?',
+            choices: [
+                {
+                    name: 'Automatically',
+                    value: 'ALWAYS_SEND'
+                },
+                {
+                    name: 'Processed in JavaScript by user',
+                    value: 'ASK_JAVASCRIPT'
+                }]
+        }]).catch((e) => {
+            console.log(`Could not determine when to enable AppCenter crashes. Error Reason - ${e.message}`);
```

Comment:
The original log message `Could not determine when to send AppCenter crashes.` sounds better to me.

Reply:
Sounds like we didn't have the message as analytics then, I copied from there.